docs/v1.X/docs/getting_started/installation.md

```diff
@@ -1,4 +1,80 @@
-# Installation
+
+# Installation methods
+
+You can install crowdsec in different ways :
+
+ - Most users [set up crowdsec's repositories](/Crowdsec/v1/getting_started/installation/#install-using-crowdsec-repository) and install from them, for ease of installation and upgrade
+ - Some users [use debian's official crowdsec packages](/Crowdsec/v1/getting_started/installation/#install-using-debian-official-packages)
+ - Some users download the DEB package directly and [install it manually](/Crowdsec/v1/getting_started/installation/#manually-install-the-debian-package)
+ - Some users download the tarball directly and [install it manually](/Crowdsec/v1/getting_started/installation/#install-from-release-tarball)
+ - Some users use the [docker hub image](https://hub.docker.com/r/crowdsecurity/crowdsec)
+ - And the most adventurous might want to [build & install from source](/Crowdsec/v1/getting_started/installation/#install-from-source)
+ - And some might even want to [build their own docker image](/Crowdsec/v1/getting_started/installation/#build-docker-image)
+ - Or use it with [docker-compose](https://github.com/crowdsecurity/example-docker-compose)
+
+
+!!! info
+    Packaging for FreeBSD and RedHat/CentOS are WIP at the time of writting. Documentation will be updated once those packages are published & functional.
+
+# Install using crowdsec repository
+
+Crowdsec distributes their own pragmatic debian packages that closely follow the developpement stream (packages are automatically published on release), and are suitable for those that want to keep up with latest changes of crowdsec.
+
+## setup the repository
+
+<details>
+  <summary>pre-requisites to add custom repositories on debian</summary>
+```bash
+sudo apt install software-properties-common
+```
+</details>
+
+```bash
+wget -qO - https://s3-eu-west-1.amazonaws.com/crowdsec.debian.pragmatic/crowdsec.asc |sudo apt-key add - && sudo apt-add-repository "https://s3-eu-west-1.amazonaws.com/crowdsec.debian.pragmatic/$(lsb_release -cs) $(lsb_release -cs) main"
+sudo apt-get update
+```
+
+The following debian suites / architectures are available :
+
+## install crowdsec
+
+```bash
+sudo apt-get install crowdsec
+```
+
+## compatbility matrix
+
+<center>
+
+| Suite       | Architecture     |
+| :------------- | :----------: | 
+|  bionic | amd64, arm64, i386   |
+| bullseye   | amd64, i386 |
+| buster | amd64, arm64, i386 |
+| focal |  amd64, arm64, i386 | 
+| sid | amd64, i386 |
+| stretch | amd64, arm64, i386 | 
+| xenial | amd64, arm64, i386 | 
+
+</center>
+
```

Comment:
Maybe add that missing build are wip too. We want to add debian packages for armhf and raspberrypios too.

Reply:
I think this is true for more advanced users